appeals/web/src/server/lib/mappers/__tests__/mappers.test.js

```diff
@@ -62,6 +34,34 @@ describe('appeal-mapper', () => {
 			const session = { account: createAccountInfo() };
 			await expect(initialiseAndMapAppealData(undefined, currentRoute, session)).rejects.toThrow();
 		});
+		it('should have an id that is unique', async () => {
+			const currentRoute = 'testroute/';
+			/**
+			 * @type {SessionWithAuth}
+			 */
+			// @ts-ignore
```

Comment:
Is the `ts-ignore` required considering there is a type definition on the line above?